test/Microsoft.Extensions.Configuration.FileExtensions.Test/FileConfigurationBuilderExtensionsTest.cs

```diff
@@ -44,11 +44,13 @@ public void GetFileProvider_ReturnPhysicalProviderWithBaseDirectoryIfNotSet()
 
             string expectedPath;
 
-#if NETCOREAPP1_1
+#if NETCOREAPP2_0
             expectedPath = AppContext.BaseDirectory;
-#else
-            expectedPath = Path.GetFullPath(AppDomain.CurrentDomain.GetData("APP_CONTEXT_BASE_DIRECTORY") as string ?? 
+#elif NET452
+            expectedPath = Path.GetFullPath(AppDomain.CurrentDomain.GetData("APP_CONTEXT_BASE_DIRECTORY") as string ??
```

Comment:
Can we use PlatformAbstractions here (https://github.com/aspnet/PlatformAbstractions/blob/dev/src/Microsoft.Extensions.PlatformAbstractions/ApplicationEnvironment.cs#L13)? Seems unnecessary to have this code here.